examples/booking_app/.gitignore

```diff
@@ -23,6 +23,7 @@
 # Flutter/Dart/Pub related
 **/doc/api/
 **/ios/Flutter/.last_build_id
+/test/**/failures/
```

Comment:
@kostadin-damyanov-prime do you think it's good idea to ignore the golden test failures in the `rx_bloc_cli` as well ?